src/OrleansVSTools/VSProjectTemplateGrainInterfaceVB/VSProjectTemplateGrainInterfaceVB.vbproj

```diff
@@ -64,6 +64,11 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="My Project\AssemblyInfo.vb" />
+    <Compile Include="Settings1.Designer.vb">
+      <AutoGen>True</AutoGen>
+      <DesignTimeSharedInput>True</DesignTimeSharedInput>
+      <DependentUpon>Settings.settings</DependentUpon>
+    </Compile>
```

Comment:
I think they were added automatically by VS as I upgraded NuGets, but I'm not 100% positive. We could do another round of cleanup across all templates and samples. I thought of preparing a separate PR for that.